src/public/Interfaces.ts

```diff
@@ -215,3 +215,31 @@ export interface Logger {
      */
     verbose(...args: any[]): void;
 }
+
+export interface Timer {
```

Comment:
We actually do not use the "master" branch. I just opened a PR to update the docs on the readme to more accurately describe the branching strategy: https://github.com/Azure/azure-functions-nodejs-worker/pull/424. Please choose either the "v1.x" or "v2.x" branch as the target for your PR. 

Personally, I would recommend the "v2.x" branch. The properties for a timer trigger changed from PascalCase to camelCase starting with v2 of the worker (v3 of the host) and it appears you are using camelCase. That change is documented [here](https://docs.microsoft.com/azure/azure-functions/functions-versions?tabs=javascript%2Cv4#breaking-changes-between-2x-and-3x).

Reply:
Regarding the versioning, especially v2 and v3, the docs are **very** unclear on the distinguish between the versions. Half of the time there is no docs for v3 at all, despite v3 not being backwards compatible with v2 in many cases.

Reply:
This is the main doc regarding versions: 
https://docs.microsoft.com/azure/azure-functions/functions-versions

With this section on breaking changes:
https://docs.microsoft.com/azure/azure-functions/functions-versions?tabs=javascript%2Cv4#breaking-changes-between-2x-and-3x

Do you have examples or suggestions of where you expect to see more docs on versioning? Would be happy to get those updated.